### PR TITLE
Fixed bug with localization

### PR DESCRIPTION
**Pull request by bikevit2008**
Old version contains bug with wrong properties for dayPicker. I fixed this. And now this works.

### CODE REVIEW

File: packages/react-jsx-highstock-datepickers/src/index.js [hunk 1]
```diff
@@ -58,6 +58,10 @@ class DateRangePickers extends Component {
       fromLabel,
       toLabel
     };
+    //For input field value
+		moment.locale(locale, {
+			...langOpts
+		});
```
Comment (whawker) [inline packages/react-jsx-highstock-datepickers/src/index.js]: Would you mind using spaces instead of tabs for consistency with the rest of the codebase?

**Comment by bikevit2008** [comment]
No problem)

**Comment by whawker** [comment]
Thanks for this. Looks like I need to do a proper overhaul of the localisation here. Seems unnecessarily complicated.

**Comment by whawker** [comment]
Just published `react-jsx-highstock-datepickers@3.0.2`. Cheers @bikevit2008!

**Comment by bikevit2008** [comment]
Yippee!

**Pull request merged**